docs/api.mjs

```diff
@@ -1,23 +1,28 @@
 // This script uses documentation to generate API Reference files
-const path = require('path');
-const documentation = require('documentation');
-const fs = require('fs');
+import { join, basename, dirname } from 'path';
+import { build, formats } from 'documentation';
+import { fileURLToPath } from 'url';
+import { existsSync, writeFileSync } from 'fs';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
 const docRoot = __dirname;
-const srcRoot = path.join(docRoot, '../src/');
-const START_EVENTS = '{START_EVENTS}';
-const END_EVENTS = '{END_EVENTS}';
-const REPLACE_EVENTS = '{REPLACE_EVENTS}';
+const srcRoot = join(docRoot, '../src/');
+const START_EVENTS = '{START\\_EVENTS}';
```

Comment:
The latest version docs escapes `_` so here we escape the escape so we can inject the events correctly.